Wiki/zh-Tw.js

```diff
@@ -0,0 +1,678 @@
+exports.LANGUAGE = {
+  CODE: "zh-Tw",
+  STANDARD_CODE: "zh-Tw",
+  FULL_CODE: "正體中文",
+  WIKI_CODE: "zh-Tw",
+  CONTRIBUTORS: ["Kami-Sqmf", "iLxlo_"]
+};
+
+exports.TIME = {
+  SECOND: "秒",
+  SECONDS: "秒",
+  MINUTE: "分",
+  MINUTES: "分",
+  HOUR: "小時",
+  HOURS: "小時",
+  DAY: "天",
+  DAYS: "天",
+  WEEK: "週",
+  WEEKS: "週",
+  MONTH: "月",
+  MONTHS: "月",
+  YEAR: "年",
+  YEARS: "年"
+};
+
+exports.PAGE_TITLES = {
+  HOME: "首頁",
+  INVITE_FILO: "邀請 Filo",
+  ANTI_BAD_WORDS: "反壞話",
+  ANTI_EVASION: "反規避",
+  ACTION_LIST: "事項清單",
+  ANTI_INVITES: "反邀請",
+  ANTI_MASS_MENTIONS: "防止提及",
+  ANTI_REPEATED_TEXT: "反重複文字",
+  AUTO_MODERATION: "AUTO MODERATION",
```

Comment:
CAPITAL LETTERS!! Sorry, there are capital letters here, remember it must be "Auto Moderation".